### PR TITLE
LPS-185599 | Test Fix | Master

### PR DESCRIPTION
**Pull request by davisaints**
**JIRA Ticket: [LPS-185599](https://issues.liferay.com/browse/LPS-185599)**

### CODE REVIEW

**Review by timpak — APPROVED** [review]
@davisaints I'm assuming you tested this.

In the future, please remember to provide context on the ticket. You want to explain the outcome of your investigation and how/why we're fixing the test.

**Comment by timpak** [comment]
ci:forward

**Comment by davisaints** [comment]
> @davisaints I'm assuming you tested this.
> 
> In the future, please remember to provide context on the ticket. You want to explain the outcome of your investigation and how/why we're fixing the test.

Yes, I tested. I will do this next time. Thanks for the reminder, @timpak

File: modules/apps/object/object-test/src/testFunctional/paths/ObjectCustomViews.path [hunk 1]
```diff
@@ -292,7 +292,7 @@
 	</tr>
 	<tr>
 		<td>FIELD_VIEW_FILTER_DATE_INPUT</td>
-		<td>//div[@class='col-lg-6'][${key_dateIndex}]//div[contains(@class, 'input-group-item')]/input[contains(@placeholder,'YYYY-MM-DD')]</td>
+		<td>//div[@class='col-lg-6'][${key_dateIndex}]//div[contains(@class, 'input-group-item')]/input[contains(@placeholder,'__/__/____')]</td>
```
Comment (davisaints) [inline modules/apps/object/object-test/src/testFunctional/paths/ObjectCustomViews.path:295]: The [DateTime](https://issues.liferay.com/browse/LPS-143068) story must have affected something in the Aggregation Filter. Previously the DD-MM-YYYY format was used. But now the month/day/year format is being used here.

**Pull request closed without merging**